README.md

```diff
@@ -92,6 +93,36 @@ The above output includes only software that is visible in the container (i.e.,
 syft <image> --scope all-layers
 ```
 
+#### Format conversion - __experimental__
```

Comment:
nit: I think the `__` use here has no effect, it's bolding what's already bold due to being a header.

Also, what's the thinking behind using H4 here?

Suggestion: `### Format conversion (experimental)`

Reply:
The neighbor sections have H4 so I followed suit.